### PR TITLE
added the key bindings for first and last selection in suggestions.

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
For #43511

### CODE REVIEW

File: src/vs/editor/contrib/suggest/suggestController.ts [hunk 1]
```diff
@@ -406,7 +406,13 @@ registerEditorCommand(new SuggestCommand({
 registerEditorCommand(new SuggestCommand({
 	id: 'selectLastSuggestion',
 	precondition: ContextKeyExpr.and(SuggestContext.Visible, SuggestContext.MultipleSuggestions),
-	handler: c => c.selectLastSuggestion()
+	handler: c => c.selectLastSuggestion(),
+	kbOpts: {
+		weight: weight,
+		kbExpr: EditorContextKeys.textFocus,
+		primary: KeyCode.End,
+		secondary: [KeyCode.End],
```
Comment (jrieken) [inline src/vs/editor/contrib/suggest/suggestController.ts]: Use `secondary` only when the key code is different from `primary` 🤓

File: src/vs/editor/contrib/suggest/suggestController.ts [hunk 2]
```diff
@@ -437,7 +443,13 @@ registerEditorCommand(new SuggestCommand({
 registerEditorCommand(new SuggestCommand({
 	id: 'selectFirstSuggestion',
 	precondition: ContextKeyExpr.and(SuggestContext.Visible, SuggestContext.MultipleSuggestions),
-	handler: c => c.selectFirstSuggestion()
+	handler: c => c.selectFirstSuggestion(),
+	kbOpts: {
+		weight: weight,
+		kbExpr: EditorContextKeys.textFocus,
+		primary: KeyCode.Home,
+		secondary: [KeyCode.Home],
```
Comment (jrieken) [inline src/vs/editor/contrib/suggest/suggestController.ts]: same

**Review by jrieken — CHANGES_REQUESTED** [review]

File: src/vs/editor/contrib/suggest/suggestController.ts (hunk 1 above)
Comment (pradeepmurugesan) [reply src/vs/editor/contrib/suggest/suggestController.ts]: Fixed

File: src/vs/editor/contrib/suggest/suggestController.ts (hunk 2 above)
Comment (pradeepmurugesan) [reply src/vs/editor/contrib/suggest/suggestController.ts]: Fixed

**Comment by pradeepmurugesan** [comment]
@jrieken sorry my bad.. Didn't know about the secondary thing. Fixed now..

**Comment by jrieken** [comment]
Thanks, looking good

**Review by jrieken — COMMENTED** [review]
👍

**Pull request merged**